src/client/components/note.vue

```diff
@@ -21,6 +21,12 @@
 			</router-link>
 		</i18n>
 		<div class="info">
+			<span v-if="isMyRenote">
+				<fa :icon="faTimes"/>
+				<span>&nbsp;</span>
+				<a href="#" @click.prevent="delRenote()">{{ $t('unrenote') }}</a>
+				<span>&nbsp;</span>
+			</span>
 			<button class="_button time" @click="showRenoteMenu()" ref="renoteTime"><mk-time :time="note.createdAt"/></button>
```

Comment:
It would be better if the change is integrated with the existing `showRenoteMenu()`.